### ISSUE TITLE
Abilty to batch rename files

### ISSUE THREAD

**Issue by rahmanshaber** [blissd/fotema#227]
After face detection/ albums/  ability to rename them in a batch would be a great feature

**Comment by blissd** [comment]
Hi @rahmanshaber. Thanks for your suggestion. Yes, I'd like to have a feature where similar faces are grouped and the user can bulk add them to a person (while also being able to exclude any miscategorised faces).

**Comment by rahmanshaber** [comment]
yes, in addition to that able to rename the files that is in a face group. this would allow to search those files using the file manager so we and mange the files in the file system
